### PR TITLE
Unit Tests of AddPeopleViewModel and YouPageViewModel 

### PR DESCRIPTION
**Pull request by abdussamadbsg**
Loom _ Free Screen Video Recording Software (2) (1)

### CODE REVIEW

**Review by pauleke65 — CHANGES_REQUESTED** [review]
Changes Requested

File: test/view_models/you_page_viewmodel_test.dart [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:hng/app/app.router.dart';
+import 'package:mockito/mockito.dart';
+import '../helpers/test_helpers.dart';
+import 'package:hng/ui/nav_pages/you_page/you_page_viewmodel.dart';
+
+void main() {
+  group('YouPageViewModelTest -', () {
```
Comment (pauleke65) [inline test/view_models/you_page_viewmodel_test.dart:8]: also only navigation write test for the user status changing from active to away and back

File: lib/models/organization_member_model.dart [hunk 2]
```diff
@@ -1,30 +1,54 @@
+class OrganizationMemberModelList {
+  final List<OrganizationMemberModel> data;
+  final int? status;
+  final String? message;
+
+  OrganizationMemberModelList(
+      {required this.data, required this.status, required this.message});
+
+  factory OrganizationMemberModelList.fromJson(Map<String, dynamic> json) {
+    // List<OrganizationMemberModel> organizationMembersList =
```
Comment (jeremiahseun) [inline lib/models/organization_member_model.dart:10]: Please remove commented codes that are not in use.

File: lib/ui/view/organization/organization_view/organization_viewmodel.dart [hunk 3]
```diff
@@ -132,15 +134,22 @@ class OrganizationViewModel extends BaseViewModel {
...
+
+      // print('Organization id from user service ${_userService.currentOrgId}');
 
       if (orgId.isNotEmpty) {
         final orgMemberList = await api.getOrganizationMemberList(orgId);
 
-        if (orgMemberList.isNotEmpty) {
-          storageService.setString(
-              StorageKeys.organizationMemberList, jsonEncode(orgMemberList));
+        // print('Organization member list length ${orgMemberList.data.length}');
```
Comment (jeremiahseun) [inline lib/ui/view/organization/organization_view/organization_viewmodel.dart:145]: Comments should be an explanation of your codes.
Commenting codes should be temporal - unless of course you will use it later in the app.

**Review by jeremiahseun — CHANGES_REQUESTED** [review]
Please make corrections

File: lib/ui/view/organization/organization_view/organization_viewmodel.dart (hunk 3 above)
Comment (abdussamadbsg) [reply lib/ui/view/organization/organization_view/organization_viewmodel.dart]: okay, thank you. changes made.

File: lib/models/organization_member_model.dart (hunk 2 above)
Comment (abdussamadbsg) [reply lib/models/organization_member_model.dart]: okay

**Review by jeremiahseun — APPROVED** [review]
LGTM!

**Review by YenumJay — APPROVED** [review]
Looks Good

**Review by pauleke65 — APPROVED** [review]
LG

**Pull request merged**